### ISSUE TITLE
[Bug] The 'href' property is missed for the CellOptions interface.

### ISSUE THREAD

**Issue by elaichenkov** [cli-table/cli-table3#320]
There's an issue with types. Because the 'href' property is missing in the interface.

Reproducible sample:

```ts
const table = new Table();
 
table.push([{ content: 'Text Link', href: 'http://example.com' }]);
```

<img width="1094" alt="image" src="https://user-images.githubusercontent.com/29764053/213209656-b3d08ea2-51f0-4237-b004-704b48747325.png">

**Comment by theoephraim** [comment]
@speedytwenty - I just installed the latest (`0.6.3`) and somehow it seems this change did not make it into what is published on npm?

You can see it is missing here: https://www.npmjs.com/package/cli-table3/v/0.6.3?activeTab=code

Thanks so much!

**Comment by speedytwenty** [comment]
> this change did not make it into what is published on npm?

@theoephraim The change occurred _after_ the 0.6.3 release. There has not been a release since.

I'd like to work #295 in with the fix but haven't had time. I'll see about getting a release out with just the fix when time permits. Cheers!

**Comment by theoephraim** [comment]
Gotcha! No worries and thanks so much for this tool :)

A full migration to typescript source might be worthwhile rather than maintaining the types. I recently undertook one for a vey old tool I maintain and it wasn't nearly as bad as I thought it would be. Lots of time saved on dealing with types and subtle bugs discovered... TS tooling is a lot better now than it used to be too.

Cheers :D

**Comment by speedytwenty** [comment]
Thanks for the input. PRs are welcome! :)

The "as bad as I think" is that the essence of cli-table3 is on a basis of clunky tests. The current documentation is generated from what seems to have started as a neat idea to have tests double as documentation code samples.

I could be overestimating it myself, but I'd probably seek to refactor the tests and documentation either in the process or before. Before seems to make sense as a matter of maintaining (or maximizing) test integrity.

**Comment by speedytwenty** [comment]
This type fix is now available in the `0.6.4` release: https://github.com/cli-table/cli-table3/releases/tag/v0.6.4
